.github/workflows/golangci-lint.yml

```diff
@@ -13,10 +13,11 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v5
         with:
-          go-version: '1.22.11'
+          go-version: '1.24'
       - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.59.1
+          version: v1.64.2
           # Verbose with color. Just fail, don't fix issues. Use both annotations and normal output.
           args: -v --color=always --fix=false --timeout=10m --out-format=colored-line-number,github-actions
+          verify: 'false'
```

Comment:
gha of golangci-lint fails to verify the configuration for v2, so skip verification.